core/src/main/scala/org/apache/spark/status/api/v1/ApiRootResource.scala

```diff
@@ -166,11 +166,11 @@ private[v1] class JsonRootResource extends UIRootFromServletContext {
 
 }
 
-private[spark] object JsonRootResource {
+private[spark] object ApiRootResource {
 
   def getJsonServlet(uiRoot: UIRoot): ServletContextHandler = {
```

Comment:
Since you changed even the class name, I'd rename this `getServletHandler` (to match its return type).